src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -212,11 +216,11 @@ public static class PublishedContentExtensions
                 if (cacheAttr != null)
                 {
                     var ctx = new DittoCacheContext(cacheAttr, content, type, culture);
-                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted));
+                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted));
                 }
                 else
                 {
-                    return ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted);
+                    return ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted);
```

Comment:
@jamiepollock I think it's fine... just wanted to flag it (probably me being pedantic late last night 😉)

@mattbrailsford You cool with this?